pkg/activitypub/store/memstore/memstore.go

```diff
@@ -0,0 +1,242 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package memstore
+
+import (
+	"fmt"
+	"net/url"
+	"sync"
+
+	"github.com/trustbloc/edge-core/pkg/log"
+
+	"github.com/trustbloc/orb/pkg/activitypub/store/spi"
```

Comment:
what's the (future) relationship between this spi and the aries-framework-go spi?

Reply:
This SP was not meant to be a generic store interface - it's specific to activities/actors. I have an issue open to have an implementation using the Aries store: https://github.com/trustbloc/orb/issues/53.